src/components/OutboundLink.js

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+var assign = require('react/lib/Object.assign');
+
+var OutboundLink = React.createClass({
+  displayName: 'OutboundLink',
+  statics: {
+    trackLink: function() {
+      console.warn("ga tracking not enabled");
+    }
+  },
+  handleClick: function (e) {
+    e.preventDefault();
+    var props = this.props;
+    OutboundLink.trackLink(props.args, function () {
+      if ( props.target === '_blank' ) {
+        window.open(props.to, '_blank');
```

Comment:
oh yea, I was gonna do that but got lazy somehow... lol. Do you think I should store props.to as a var as well?